app/src/main/java/com/pajato/android/gamechat/exp/CheckersFragment.java

```diff
@@ -61,7 +62,10 @@
     public static final String SECONDARY_PIECE = "sp";
     public static final String SECONDARY_KING = "sk";
 
-    public ImageButton mHighlightedTile;
+    public static final String KING_UNICODE = "\u26c1";
```

Comment:
There is a substantial gain to be had in reducing code by doing something along the lines of:
`public enum Piece {
     primary('\u26c1', R.color.red),
     secondary('\uxxxx', R.color.black),
     ....
     secondaryKing('\uxxxx', R.color.black);
`